### DISCUSSION TITLE
Termux and Android 13, any issues?

### DISCUSSION THREAD

**Discussion by macerg** [termux/termux-packages#19510] [Q&A] [answered]
Hello, I've been running termux on a number of devices, included rooted android phones. I love it and depend on it for many reasons. I'm thinking of moving to a Fairphone 5 (rooted), are there any issues I am likely to encounter? Android 13?

**Comment by macerg** [comment]
I guess there's no issues then?

**Reply by agnostic-apollo** [reply, marked as answer]
No known issue other than phantom process killer for A13 and you have root so can more easily disable it.

**Comment by macerg** [comment]
How do you disable the phantom process killer please?

**Reply by agnostic-apollo** [reply]
God gosh, search or look into termux-app repo README.

**Comment by macerg** [comment]
Check out my contribution history. Thanks for nothing. Prick.

**Reply by truboxl** [reply]
I recommend keep submitting bug reports. If there's any issue, maintainers will try to resolve ASAP. Thanks. Admitedly we lack testing so we cannot provide all the answers except through trials and errors.
